### PR TITLE
Format the coalesce expression we produce.

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**
Fixes https://github.com/dotnet/roslyn/issues/16025

### CODE REVIEW

**Comment by CyrusNajmabadi** [comment]
Tagging @dotnet/roslyn-ide

File: src/EditorFeatures/CSharpTest/UseCoalesceExpression/UseCoalesceExpressionTests.cs [hunk 1]
```diff
@@ -310,5 +310,34 @@ void M(string x, string y, string z)
...
+}",
+@"using System;
+
+class Program
+{
+    public Program()
+    {
+        string x = "";
+
+        string y = x ?? string.Empty;
```
Comment (Pilchie) [inline src/EditorFeatures/CSharpTest/UseCoalesceExpression/UseCoalesceExpressionTests.cs]: What was different before this?

Comment (CyrusNajmabadi) [inline src/EditorFeatures/CSharpTest/UseCoalesceExpression/UseCoalesceExpressionTests.cs]: You could have trailing trivia between the false part and the semicolon

**Comment by CyrusNajmabadi** [comment]
Tagging @srivatsn this week fix an annoying formatting issue. This is also especially important for people who do fix-all. !

**Pull request merged**